tests/test_response.py

```diff
@@ -54,3 +67,13 @@ def test_view_can_set_own_status_code(self):
 
         self.response.view(self.app.resolve(ControllerTest().change_status))
         assert self.request.is_status(203)
+
+    def test_view_should_return_a_json_response_when_retrieve_a_user_from_model(self):
+        
+        assert isinstance(MockUser(), Model)
+
+        self.response.view(MockUser().find(1))
+
+        json_response = [{'name':'TestUser', 'email': 'user@email.com' }]
+
+        assert self.app.make('Response') == json_response
```

Comment:
this test is good but not correct. 

```python
json_response = "[{'name':'TestUser', 'email': 'user@email.com' }]"
assert self.app.make('Response') == json_response
```

that should be the correct assertion.

Reply:
since it needs to be a string when going to the browser

Reply:
That change in the response class above should fix that.